### PR TITLE
Load database config, docker fix

### PR DESCRIPTION
**Pull request by markspolakovs**
@BabarKhanDev Is this the right way of doing for both local and Docker? `load_database_config`'s parameters are `filename=os.getenv('CONFIG_FILE', "../config.ini")`, so that should work locally if CONFIG_FILE isn't set, right?

### CODE REVIEW

**Comment by BabarKhanDev** [comment]
This won't work because when we run uvicorn it's from the base of the repo.
We could change the signature to:
```python
def load_database_config(filename=os.getenv('CONFIG_FILE', "config.ini")):
```
That might cause issues for data_generators.py but we can scrap that and use the endpoint if we want

**Comment by markspolakovs** [comment]
Thanks, updated.

File: util/data.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class ProcessedEntry(BaseModel):
 # Database entries end
 
 
-def load_plant_net_api_key(filename=os.getenv('CONFIG_FILE', "../config.ini")):
+def load_plant_net_api_key(filename=os.getenv('CONFIG_FILE', "config.ini")):
```
Comment (BabarKhanDev) [inline util/data.py:23]: I think this also needs changing for load_database_config()

**Review by BabarKhanDev — CHANGES_REQUESTED** [review]

**Review by BabarKhanDev — APPROVED** [review]
lgtm :)

**Pull request merged**